shuttle/homepage/homepage_controller.py

```diff
@@ -12,6 +12,11 @@ def __init__(self):
 
     def grab_check_in_driver_data(self):
         data = db.get_last_location()
+        now = datetime.now()
+        current_date = now.strftime('%b-%d-%y')
```

Comment:
Could simplify this to just one line if you wanted 

current_date = datetime.now().strftime('%b-%d-%y')